libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/SPRequestHandler.java

```diff
@@ -63,9 +63,20 @@
      * @param authCallback Auth callback.
      */
     public SPRequestHandler(String loginUrl, LoginActivity.SPAuthCallback authCallback) {
+        this(loginUrl, null, authCallback);
+    }
+
+    /**
+     * Parameterized constructor.
+     *
+     * @param loginUrl Login URL.
+     * @param userHint User hint. Must be of the format 'orgId:userId', both being 18-char IDs.
+     * @param authCallback Auth callback.
+     */
+    public SPRequestHandler(String loginUrl, String userHint, LoginActivity.SPAuthCallback authCallback) {
```

Comment:
Added a convenience constructor to pass in `user_hint`. Need this for IDP initiated flow.